addons/account_peppol/wizard/account_move_send_batch_wizard.py

```diff
@@ -4,6 +4,14 @@
 class AccountMoveSendBatchWizard(models.TransientModel):
     _inherit = 'account.move.send.batch.wizard'
 
+    def _compute_summary_data(self):
+        # EXTENDS 'account' - add checking of partner's validity
+        for wizard in self:
+            if peppol_moves := wizard.move_ids.filtered(lambda m: wizard._get_default_sending_method(m) == 'peppol'):
+                for move in peppol_moves:
+                    move.partner_id.commercial_partner_id.button_account_peppol_check_partner_endpoint(company=move.company_id)
```

Comment:
Weird to have this call in a compute...  Is there a better alternative?

Reply:
Not really, it's a discussion we already had on the "single" wizard counterpart where we had to do the same.
The default_get is not really useful for that. Only alternative is to make sure before creating(/new) each wizard we do it, but obviously that not a very robust choice...